### DISCUSSION TITLE
Default the proxy setting for Drivers.

### DISCUSSION THREAD

**Discussion by RickSaavedra** [dbeaver/dbeaver#14861] [Q&A]
#### System information: 
- Windows 10
- 21.3.0
- 11.0.12+7

#### Describe the problem you're observing:

We are trying to default the install to include default proxy information for drivers to download. Can this be setup somehow using i.e. (ini file, command line options)?

**Comment by ShadelessFox** [comment]
Hello @RickSaavedra,

Proxy settings, among others, are stored within the workspace folder.

Take a look at https://github.com/dbeaver/dbeaver/wiki/Workspace-Location.

**Reply by RickSaavedra** [reply]
Hi @ShadelessFox ,

How could I set it up so whenever a new user starts Dbeaver for the first time the Driver Proxy settings would be defaulted? Is that possible?

Thanks

**Reply by ShadelessFox** [reply]
It's possible by distributing workspace along with DBeaver.
